backend/server/src/main/java/com/issuetracker/dto/ios/IosNewIssueDto.java

```diff
@@ -0,0 +1,68 @@
+package com.issuetracker.dto.ios;
+
+import com.issuetracker.domain.NewIssue;
+
+public class IosNewIssueDto {
+
+    private String title;
+
+    private String comment;
+
+    private Long[] assigneesId;
+
+    private Long[] labelsId;
```

Comment:
```suggestion
    private List<UserDto> assignees;

    private List<LabelDto> labels;
```

프론트는 화면에 출력할 용도로 데이터를 받기를 원할 텐데, id 값만 넘겨주는게 프론트 분들께 유용할지 의문입니다.
줄거면 Dto 들의 리스트를 넘겨주어야지요.
여기서 domain 과 dto 의 차이점을 한가지 더 알 수 있는데,
dto 는 Jackson 을 사용한 변환을 쉽게하기 위해 List 의 사용을 권장하지만,
domain 은 이런 경우 따로 일급 컬렉션 객체를 만드는 것을 권장합니다.

Reply:
https://github.com/ghojeong/issue-tracker/commit/c78c6164429820f4256bf80ea420fa76c630d4c3
수정 완료했습니다! ㅎㅎ